fenics/Dockerfile

```
@@ -0,0 +1,32 @@
+ARG ROOT_CONTAINER=pyiron/base:latest
+ARG BASE_CONTAINER=$ROOT_CONTAINER
+FROM $BASE_CONTAINER
+
+MAINTAINER Jan Janssen <janssen@mpie.de>, Muhammad Hassani <hassani@mpie.de>
+
+USER $DOCKER_UID
+WORKDIR $HOME
+ARG PYTHON_VERSION=default
+
+ENV OMPI_MCA_plm=isolated
+ENV OMPI_MCA_rmaps_base_oversubscribe=yes
+ENV OMPI_MCA_btl_vader_single_copy_mechanism=none
+
+COPY . ${HOME}/
+RUN conda env update -n base -f ${HOME}/environment.yml && \
+    conda clean --all -f -y && \
+    npm cache clean --force && \
```

```suggestion
    npm cache clean --force && \
```
As `nglview` is not installed I do not think we need this part.

---

Done!